source/NVDAObjects/IAccessible/sysListView32.py

```diff
@@ -70,7 +70,10 @@
 LVCF_IMAGE=16
 LVCF_ORDER=32
 
-CBEMAXSTRLEN=260

```

Comment:
I wonder if it was 256 + 4 bytes (4 bytes for \0 maybe?) and if 516 would then not be more enjoyable.